fladle-plugin/src/main/java/com/osacky/flank/gradle/YamlWriter.kt

```diff
@@ -83,6 +83,9 @@ internal class YamlWriter {
     appendProperty(config.recordVideo, name = "record-video")
     appendProperty(config.performanceMetrics, name = "performance-metrics")
     appendProperty(config.testTimeout, name = "timeout")
+    if (config.async.isPresent && config.async.get()) {
```

Comment:
can the existing `appendProperty` method be used here?

Reply:
I tried using `appendProperty()` directly without checking the new property's presence and value, but it looks like there are tests that assert on the output yml and broke when `async` was added with a default value of `false`.

Instead of changing those tests to handle the async flag, I only add the `async` flag when it's explicitly added as in the `fladle {}` block.

Reply:
interesting. i guess this was written in that way :\